app/containers/Questions/QuestionFilter.js

```diff
@@ -60,61 +60,64 @@ const QuestionFilter = ({
   changeQuestionFilterDispatch,
   questionFilterFromCookies,
 }) => {
-  if (!single) return null;
-
-  const [filter, setFilterValue] = useState(+(questionFilterFromCookies || 1));
-
-  useEffect(
-    () => {
-      const cookieValue = questionFilterFromCookies;
-
-      setFilterValue(cookieValue);
-
-      if (!cookieValue || cookieValue === '1') {
-        setCookie(cookieFilterSetter(1));
-        changeQuestionFilterDispatch(1);
-      }
-    },
-    [filter],
-  );
-
-  const setFilter = useCallback(
-    value => {
-      if (value !== filter) {
-        setFilterValue(value);
-        setCookie(cookieFilterSetter(value));
-        changeQuestionFilterDispatch(value);
-      }
-    },
-    [filter],
-  );
-
-  const setAllFilter = useCallback(
-    () => {
-      setFilter(0);
-      setCookie({
-        name: UPDATE_PROMO_QUESTIONS,
-        value: true,
-      });
-    },
-    [filter],
-  );
-  const setQuestionFilter = useCallback(() => setFilter(1), [filter]);
-
-  return display ? (
-    <Container>
-      <Button
-        active={questionFilterFromCookies === '1'}
-        onClick={setQuestionFilter}
-        left
-      >
-        <FormattedMessage {...commonMessages.top} />
-      </Button>
-      <Button active={questionFilterFromCookies === '0'} onClick={setAllFilter}>
-        <FormattedMessage {...commonMessages.all} />
-      </Button>
-    </Container>
-  ) : null;
+  return null;
+  //TODO Question Filter
+  // if (!single) return null;
+
+  // const [filter, setFilterValue] = useState(+(questionFilterFromCookies || 1));
+
+  // useEffect(
+  //   () => {
+  //     const cookieValue = questionFilterFromCookies;
+
+  //     setFilterValue(cookieValue);
+
+  //     if (!cookieValue || cookieValue === '1') {
+  //       setCookie(cookieFilterSetter(1));
+  //       changeQuestionFilterDispatch(1);
+  //     }
+  //   },
+  //   [filter],
+  // );
+  // // console.log('nonon1', cookieValue); 
```

Comment:
remove `console.log` and add `TODO:` why this code was commented
<img width="468" alt="image" src="https://user-images.githubusercontent.com/28437795/160828483-cb9be984-9d45-461e-af40-832e746dd878.png">